build.js

```diff
@@ -422,655 +699,812 @@ function buildSupportProjects(api, config) {
 
       return spawnWithLogger(api, 'ndk-build', args , {cwd: tealeafDir});
     });
-}
+}*/
```

Comment:
this function that is commented out, can be removed if it isn't used

Reply:
Done